### ISSUE TITLE
A user is unable to fold the code in the editor using keyboard

### ISSUE THREAD

**Issue by LadyNaggaga** [microsoft/monaco-editor#1168]
.NET Online REPL  is having the following accessibility issue.  Please see this [issue](https://github.com/dotnet/try/issues/121) . I have also included the details below 
> **Environment Details**
> https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> Browser: Edge (41.16299.402.0)
> Chrome (67.0.3396.87)
> 
> **Additional Details:**
> MAS Violated: MAS2.1.1
> 
> **Repro steps:**
> Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> Tab till “Enter focus mode” and press enter
> Write a Sample program
> Now press F1 and tab till Fold and press enter
> Verify Fold action is getting performed or not.
> 
> **Actual Result:**
> A user is unable to fold the code in the editor using keyboard
> 
> **Expected Result:**
> A user should be able to fold the code using either shortcut or navigating to Fold in command Palette
> 
> **User Impact:**
> A disabled user wouldn't be able to avail the functionality.
> 
> **MAS Reference:**
> [https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D)
> 
> [A User is unable to fold the code in the editor.pptx](https://github.com/dotnet/try/files/2191629/A.User.is.unable.to.fold.the.code.in.the.editor.pptx)

**Comment by terrimorton** [comment]
This functionality is actually now present for keyboard users. Upon entering the code editor, the screen reader (NVDA and Narrator) announces to press Alt+F1 for accessibility options. That leads to lots of accessibility help including how to open the Command Palette (F1 in Chrome/Edge and Alt+F1 in IE). In the command palette there many several keyboard shortcuts for folding and unfolding.

**Comment by terrimorton** [comment]
I think this ticket needs to be closed.
